src/tiledimage.js

```diff
@@ -685,6 +694,23 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
+     * Get the current rotation of this tiled image in degrees.
+     * @returns {Number} the current rotation of this tiled image in degrees.
+     */
+    getRotation: function() {
+        return this.degrees;
+    },
+
+    /**
+     * Set the current rotation of this tiled image in degrees.
+     * @param {Number} the rotation in degrees.
+     */
+    setRotation: function(degrees) {
+        this.degrees = $.positiveModulo(degrees, 360);
+        this._needsDraw = true;
```

Comment:
Indeed! I spotted the issue with `setClip` when writing those lines and then forgot about it :D